governance-exercises/JanSky Polling Exercise 4 - Forum Post.md

```diff
@@ -0,0 +1,20 @@
+# Forum Response
+
+Author: JanSky  
+Date: Wednesday 9 August 2023
+
+---
+
+This situation requires an immediate response. A couple of recommendations that should happen promptly:
+
+**Emergency Poll on Fix A**: I will be initiating an emergency poll presently to see if there is community support for Fix A. **This poll will only last 12 hours, @Aligned Delegates please vote as soon as you are ready and able.** While [MIP24](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) states that a poll might not be strictly necessary in emergencies, I am recommending it given the difference of opinion between Roundtable and WeCodeCrypto.
```

Comment:
You will also need to be getting in touch with the Aligned delegates in this situation.